### PR TITLE
Added Authentication AAD setup README.

### PR DESCRIPTION
**Pull request by valentinyanakiev**
### Describe the background of your pull request

What does your pull request do? Does it solve a bug (which one?), add a feature?

### Additional context

Add any other context

### Governance 

- [x] Documentation is added
- [x] I've read the contributing document https://github.com/cherrytwist/.github/blob/master/CONTRIBUTING.md
- [x] I've read and understand the Code of Conduct https://github.com/cherrytwist/.github/blob/master/CODE_OF_CONDUCT.md
- [x] I understand that any contributions or suggestions I made may make it into the actual code. I've read the License 
     https://github.com/cherrytwist/Coordination/blob/master/LICENSE 
     and the contributor license agreement: tba

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -23,7 +23,95 @@ To start using CherryTwist one needs to spin up a kubernetes cluster. Please loo
...
+   - In the **Name** section, enter a meaningful application name that will be displayed to users of the app, for example `cherrytwist-web`.
+   - Change **Supported account types** to **Accounts in any organizational directory and personal Microsoft accounts (e.g. Skype, Xbox, Outlook.com)**.
+   - Select **Register** to create the application.
+      ![App Registration 3](diagrams/authentication/App-Registration-3.png)
+1. On the app **Overview** page, find the **Application (client) ID** value and record it for later. You'll need it to configure the configuration file for this projects. It will be used as the **REACT_APP_AUTH_CLIENT_ID** environment variable value in the Cherrytwist Client App. Record the **Directory (tenant) ID**. It will be used as the **REACT_APP_AUTH_TENANT_ID** environment variable in the Cherrytwist Client App.
+      ![App Overview](diagrams/authentication/Cherrytwist-Web.png)
+1. From the app's Overview page, select the **Authentication** section.
+   - Click **Add a platform** button.
+   - Select **Single-page Applications** on the right blade.
+   - Add a **Redirect URIs**, for instance `http://localhost:3000`.
```
Comment (GhostOnTheFiber) [inline README.md:43]: Maybe also add as en example a url: 'dev.cherrytwsit.org' or something like "root url". I think people might get confused with the localhost + port and guess that it needs to be something local.

**Review by GhostOnTheFiber — APPROVED** [review]
Over all very nice work :100: One small comment but if you dont agree then merge it :rocket:

**Review by techsmyth — APPROVED** [review]
Very thorough! Would be good to also have Anton also follow it on a clean environment.

File: README.md (hunk 1 above)
Comment (valentinyanakiev) [reply README.md:43]: > Maybe also add as en example a url: 'dev.cherrytwsit.org' or something like "root url". I think people might get confused with the localhost + port and guess that it needs to be something local.

Fixed :)

**Pull request merged**